packages/tools/devtools/devtools-view/src/components/ContainerSummaryView.tsx

```diff
@@ -398,7 +398,11 @@ function ActionsBar(props: ActionsBarProps): React.ReactElement {
 			size="small"
 			icon={<PlugConnected20Regular />}
 			onClick={tryConnect}
-			disabled={tryConnect === undefined || isContainerClosed}
+			disabled={
+				tryConnect === undefined ||
+				containerState.closed ||
+				containerState.attachState === "Detached"
```

Comment:
Let's use the `AttachState` enum instead of hardcoded strings, here and on line 419.